config/longrun_configs/amip_target_topo_diagedmf_gpu.yml

```diff
@@ -16,7 +16,7 @@ netcdf_output_at_levels: true
 output_default_diagnostics: true
 start_date: "20100101"
 surface_setup: "PrescribedSurface"
-t_end: "366days"
+t_end: "300days"
```

Comment:
did you mean to include this change too?

Reply:
Yes, the longrun has 1 day walltime limit and we can get about 300 days with the current SYPD.